Cargo.toml

```diff
@@ -13,15 +13,15 @@ bitflags = "2.0"
 byteorder = "1.3.2"
 cfg-if = "1.0"
 crash-context = "0.6"
```

Comment:
You just need to bump crash-context to `0.6.1` and we're done, thank you!

Reply:
0.6.1 is semver compatible with 0.6, I tend to leave off compatible versions since it just leads to churn in the manifest file, but I'll add it for clarity.